docs/balena-cli.md

```diff
@@ -1184,14 +1191,23 @@ fleet name or slug (preferred)
 
 ### Options
 
-## release &#60;commitOrId&#62;
+#### -j, --json
+
+produce JSON output instead of tabular output
 
+## release &#60;commitOrId&#62;
 
+The --json option is recommended when scripting the output of this command,
+because field names are less likely to change in JSON format and because it
+better represents data types like arrays, empty strings and null values.
+The 'jq' utility may be helpful for querying JSON fields in shell scripts
+(https://stedolan.github.io/jq/manual/).
 
 Examples:
 
 	$ balena release a777f7345fe3d655c1c981aa642e5555
 	$ balena release 1234567
+	$ balena release d3f3151f5ad25ca6b070aa4d08296aca --json
```

Comment:
Hello @bbugh the CI is failing because this was added without adding it to the examples & description on `release/index.ts` could you please ensure you make 
```ts
import { jsonInfo } from '../../utils/messages';
...
	public static description = stripIndent`
		Get info for a release.

		${jsonInfo.split('\n').join('\n\t\t')}
`;
	public static examples = [
		'$ balena release a777f7345fe3d655c1c981aa642e5555',
		'$ balena release 1234567',
		'$ balena release a777f7345fe3d655c1c981aa642e5555 --json'
	];
```
On that file and then run `npm run build && npm run catch-uncommitted` before pushing?

Reply:
I am sorry the CLI CI is a bit tricky the first time and there are a few gotchas we tried to document our best, testing locally this should be the last one of them.

Reply:
> I am sorry the CLI CI is a bit tricky the first time and there are a few gotchas we tried to document our best, testing locally this should be the last one of them.

It's not a problem at all, y'all have so many open source projects I'm sure it's a lot to keep track of. I'm really impressed at how actively response y'all are too.

> the CI is failing because this was added without adding it to the examples & description on release/index.ts

very cool that this examples thing is automated. I have made the requested change.

I haven't pushed yet though because when I ran `catch-uncommitted`, the only change it makes is removing the --json help text from the `release` command. Not sure if that's intentional, since it's the same as all of the other commands that implement this option, so I'm unsure how to proceed.

What do you advise?

Here is the full diff of `npm run build && npm run catch-uncommitted`:

```diff
diff --git a/docs/balena-cli.md b/docs/balena-cli.md
index 65c167d4..beda20eb 100644
--- a/docs/balena-cli.md
+++ b/docs/balena-cli.md
@@ -1197,11 +1197,7 @@ produce JSON output instead of tabular output
 
 ## release &#60;commitOrId&#62;
 
-The --json option is recommended when scripting the output of this command,
-because field names are less likely to change in JSON format and because it
-better represents data types like arrays, empty strings and null values.
-The 'jq' utility may be helpful for querying JSON fields in shell scripts
-(https://stedolan.github.io/jq/manual/).
+
 
 Examples:
```

Reply:
@bbugh are you 100% sure you added the `jsonInfo.split...` thing in your description? Here is my exact `release/index.ts` and there is no diff from catch-uncommitted:

```ts
/**
 * @license
 * Copyright 2016-2020 Balena Ltd.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

import { Flags, Args, type Interfaces } from '@oclif/core';
import Command from '../../command';
import * as cf from '../../utils/common-flags';
import { getBalenaSdk, getVisuals, stripIndent } from '../../utils/lazy';
import type * as BalenaSdk from 'balena-sdk';
import jsyaml = require('js-yaml');
import { tryAsInteger } from '../../utils/validation';
import { jsonInfo } from '../../utils/messages';

export const commitOrIdArg = Args.custom({
	parse: async (commitOrId: string) => tryAsInteger(commitOrId),
});

type FlagsDef = Interfaces.InferredFlags<typeof ReleaseCmd.flags>;

export default class ReleaseCmd extends Command {
	public static description = stripIndent`
		Get info for a release.

		${jsonInfo.split('\n').join('\n\t\t')}
`;
	public static examples = [
		'$ balena release a777f7345fe3d655c1c981aa642e5555',
		'$ balena release 1234567',
		'$ balena release a777f7345fe3d655c1c981aa642e5555 --json',
	];

	public static usage = 'release <commitOrId>';

	public static flags = {
		json: cf.json,
		help: cf.help,
		composition: Flags.boolean({
			default: false,
			char: 'c',
			description: 'Return the release composition',
		}),
	};

	public static args = {
		commitOrId: commitOrIdArg({
			description: 'the commit or ID of the release to get information',
			required: true,
		}),
	};

	public static authenticated = true;

	public async run() {
		const { args: params, flags: options } = await this.parse(ReleaseCmd);

		const balena = getBalenaSdk();
		if (options.composition) {
			await this.showComposition(params.commitOrId, balena);
		} else {
			await this.showReleaseInfo(params.commitOrId, balena, options);
		}
	}

	async showComposition(
		commitOrId: string | number,
		balena: BalenaSdk.BalenaSDK,
	) {
		const release = await balena.models.release.get(commitOrId, {
			$select: 'composition',
		});

		console.log(jsyaml.dump(release.composition));
	}

	async showReleaseInfo(
		commitOrId: string | number,
		balena: BalenaSdk.BalenaSDK,
		options: FlagsDef,
	) {
		const fields: Array<keyof BalenaSdk.Release> = [
			'id',
			'commit',
			'created_at',
			'status',
			'semver',
			'is_final',
			'build_log',
			'start_timestamp',
			'end_timestamp',
		];

		const release = await balena.models.release.get(commitOrId, {
			...(!options.json && { $select: fields }),
			$expand: {
				release_tag: {
					$select: ['tag_key', 'value'],
				},
			},
		});

		if (options.json) {
			console.log(JSON.stringify(release, null, 4));
		} else {
			const tagStr = release
				.release_tag!.map((t) => `${t.tag_key}=${t.value}`)
				.join('\n');

			const _ = await import('lodash');
			const values = _.mapValues(
				release,
				(val) => val ?? 'N/a',
			) as Dictionary<string>;
			values['tags'] = tagStr;

			console.log(getVisuals().table.vertical(values, [...fields, 'tags']));
		}
	}
}

```

Reply:
Correct, there is no difference in `release/index.ts` from `catch-uncommitted`. The diff I shared is in `docs/balena-cli.md`.

Reply:
I force pushed again with the added example in `release/index.ts` so that you can run the `npm run build && npm run catch-uncommitted` command and test for yourself, since I don't know what is expected and correct. :)

Reply:
@bbugh You are just missing the import and description on the `release/index.ts`:

Yours is:
```ts
	public static description = stripIndent`
		Get info for a release.
`;
```

And you need the description as

```ts
import { jsonInfo } from '../../utils/messages';
...
	public static description = stripIndent`
		Get info for a release.

		${jsonInfo.split('\n').join('\n\t\t')}
`;
```
Hope this helps

Reply:
Ah, of course! It's generated in the command too. 🤦🏻‍♂️

Thanks for your patience explaining that, I understand now what you meant before about not having a diff on `release/index.ts`. 

I fixed it and pushed again.